DAS/visualizer.py

```diff
@@ -37,25 +42,31 @@ def plottingData(self):
                 bwUplink1 = int(root.find('bwUplink1').text)
                 bwUplink2 = int(root.find('bwUplink2').text)
                 tta = int(root.find('tta').text)
+                if tta == -1:
+                    tta = self.maxTTA
```

Comment:
I think maxTTA is the wrong approach here. It is better to use NaN, and 
- an average that discards NaNs
- a percentage of results that are not NaNs
It is a statistics of two numbers instead of one, but unfortunately there is no meaningful way of providing a statistics that expresses the whole distribution with one number.  When interpreting results, you should always look at the whole distribution, at these two numbers, or at some quantiles anyway.